src/applications/vaos/new-appointment/components/calendar/CalendarWidget.jsx

```diff
@@ -255,7 +253,6 @@ export default function CalendarWidget({
   maxDate,
   maxSelections = 1,
   minDate,
-  monthsToShowAtOnce = 1,
```

Comment:
My argument for removing this is:

1. We can't test this through the date pages anymore
2. Switching to 1 month introduced some minor bugs that we can to account for, so we know it's tricky to maintain
3. So, it seems unlikely that we'll be able to reliably keep this working without it being in active use.